Project.toml

```
@@ -1,14 +1,10 @@
 name = "QEDcore"
 uuid = "35dc0263-cb5f-4c33-a114-1d7f54ab753e"
-authors = [
-    "Uwe Hernandez Acosta <u.hernandez@hzdr.de>",
-    "Anton Reinhard <a.reinhard@hzdr.de>",
-]
+authors = ["Uwe Hernandez Acosta <u.hernandez@hzdr.de>", "Anton Reinhard <a.reinhard@hzdr.de>"]
```

```suggestion
authors = ["Uwe Hernandez Acosta <u.hernandez@hzdr.de>", "Anton Reinhard"]
```
Oh i believe we wanted to delete my e-mail from this

---

Sure. Done in https://github.com/QEDjl-project/QEDcore.jl/pull/45/commits/085e2933b528afdc001d5ecec1ed7c58a32cf49a